CMakeLists.txt

```diff
@@ -473,6 +414,7 @@ endif()
 # ------------------------------------------------
 # install
 
+# TODO: the use of glob here is almost certainly brittle if not outright wrong.
```

Comment:
These are not build deps. How is globbing more brittle that explicitly listing the files, and then forgetting to update the list when a new HalideRuntimeFoo.h is added?

Reply:
Because file(GLOB) runs at build generation time and the files in "${CMAKE_BINARY_DIR}/include/" don't exist until after the build has run. I'm actually surprised this works at all.

Reply:
Ooh, I didn't catch that. We should just list them out here then, if that's the path we're going down (or at least update the comment it include what you said).

Reply:
Ah, apparently the CMakeLists.txt in src/ copies them to the build output directory at configure time. There's no reason to do this. We can issue the relevant install directives directly from there, remove the glob here, and end up with less CMake code overall.

Reply:
> I add new tests frequently, and don't use the cmake build

As a thought experiment: what would it take to get all the Makefile holdouts (myself included) to commit to using CMake regularly (or at least to get in the habit of building it before PRs)?

Reply:
Maybe switching the buildbots to it?